sympy/solvers/ode/single.py

```diff
@@ -359,6 +358,137 @@ def inverse(self):
         return diffcls
 
 
+class FirstExact(SingleODESolver):
+    r"""Solves 1st order exact ordinary differential equations.
+    A 1st order differential equation is called exact if it is the total
+    differential of a function. That is, the differential equation:
```

Comment:
There already is a docstring for this method in `ode_1st_exact`. The `ode_1st_exact` function should be removed but the docstring can be moved here